tests/sirv.js

```diff
@@ -1032,6 +1032,21 @@ ranges('should throw `416` when range cannot be met (overflow)', async () => {
 	}
 });
 
+ranges('should shrink range end is it cannot be met (overflow)', async () => {
```

Comment:
I'm guessing you meant "if" here?
```suggestion
ranges('should shrink range end if it cannot be met (overflow)', async () => {
```